### ISSUE TITLE
#VOICE ERROR 

### ISSUE THREAD

**Issue by Suchir2004** [ProgrammingHero1/romantic-alexa#124]
Traceback (most recent call last):
  File "C:\Users\HP-PC\Desktop\test.py", line 11, in <module>
    engine.setProperty('voice', voices[1].id)
IndexError: list index out of range

I install all possible pip and also pipwin, pyaudio, but it didn't resolve.
       Tell me what should I do know?

**Comment by jromerooo2** [comment]
yeah, the basic problem is that it still running while commands are executing so it kinda gets confuses,
please read my issue, that will help to solve some problems in the repo I provide
https://github.com/ProgrammingHero1/romantic-alexa/issues/127
